app/views/communities/show.html.slim

```diff
@@ -4,34 +4,39 @@ span
     = @community.title
     - if @community.closed
       = "[closed]"
-    span
-      - if @community.closed == false
-        - if policy(@community).update?
-          = link_to "Edit", edit_community_path(@community.id), class: "btn btn-primary btn-sm"
-        - if policy(@community).destroy?
-          = link_to "Close", community_path(@community.id), method: :delete, class: "btn btn-danger btn-sm", data: { confirm: "Are you sure?" }
-  span
+  .d-flex.flex-row-reverse
+    - if !@community.closed
+      - if policy(@community).update?
+        = link_to edit_community_path(@community.id), class: "action-button" do
+          i.bi.bi-pencil
+      - if policy(@community).destroy?
+        = link_to community_path(@community.id), method: :delete, class: "action-button", data: { confirm: "Are you sure?" } do
+          i.bi.bi-trash
+  span.d-flex.justify-content-center
     - @community.tag_references.each do |tag_reference|
         = link_to tag_reference.tag.title, tag_path(tag_reference.tag), class: "badge rounded-pill bg-secondary"
-hr
-- if policy(Post.new(user: current_user, community: @community)).new?
-  = link_to "New Post", new_community_post_path(@community.id), class: "btn btn-primary"
-- @posts.each do |post|
-  .card.post-card.mb-3
-    .row.g-0
-      - if post.banner.present?
-        = image_tag post.banner, class: "col-md-2"
-      .col-md-8
+.row
+  - if policy(Post.new(user: current_user, community: @community)).new?
+    .col-12.col-sm-6.col-lg-4.col-xl-3
+      .card.post-card.btn.btn-primary
+        .card-body.d-flex.justify-content-center.align-items-center
+          h1 +
+          = link_to "", new_community_post_path(@community), class: "stretched-link"
```

Comment:
kas tas par linku bez teksta? CSS klase tipa uzliek kkādu background ikonu?

Reply:
ta ir bootstrap klase kas padara taka visu to elementu ka linku, more information: 
https://getbootstrap.com/docs/5.0/helpers/stretched-link/

Reply:
Pameginaju, met ara URL uz kuru iet, tapec paliku pie empty stringa